README.md

```diff
@@ -141,7 +141,8 @@ var api = new ParseServer({
   appId: 'myAppId',
   masterKey: 'myMasterKey', // Keep this key secret!
   fileKey: 'optionalFileKey',
-  serverURL: 'http://localhost:1337/parse' // Don't forget to change to https if needed
+  serverURL: 'http://localhost:1337/parse', // Don't forget to change to https if needed
```

Comment:
Should be set only to HTTP if the server is created with https. I would still encourage to create 2 servers, 

1 http server for localhost connections
1 https server for other connections